app/src/main/java/com/pvp/app/ui/screen/statistics/StatisticsScreen.kt

```diff
@@ -335,12 +356,136 @@ private fun GraphsPast(
     )
 }
 
+@Composable
+fun StatisticsContainers(state: StatisticsState) {
+    val localeAverageTasksCompleted = stringResource(R.string.statistics_average_tasks_completed)
+    val localeAveragePoints = stringResource(R.string.statistics_average_points)
+    val localeTop3FrequentActivities = stringResource(R.string.statistics_top3_frequent_activities)
+    val localeUniqueActivities = stringResource(R.string.statistics_unique_activities)
```

Comment:
Not required, but I think you could use stringResources straightly in label parameters, e.g.:
```kotlin
label = stringResource(R.string.statistics_average_tasks_completed) + " (7d):"
```